docs/src/terminology.md

```diff
@@ -81,7 +92,7 @@ The wallclock duration between a [leader](#leader) creating a [tick entry](#tick
 
 ## confirmed block
 
-A [block](#block) that has received a [super majority](#supermajority) of [ledger votes](#ledger-vote).
+See [optimistically confirmed block](#optimistically-confirmed-block).
```

Comment:
Do we need this separate section from `optimistically confirmed block`?  Seems like we should only need one and just use the terminology throughout this doc consistently.